
try{
    let obj=undefined;
    console.log(obj.name)
}catch(err){            //err holds info about the errors
    console.log("Exception ")
    console.log(err)
}